stable/airflow/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 description: Airflow is a platform to programmatically author, schedule and monitor workflows
 name: airflow
-version: 0.17.4
+version: 0.18.0
```

Comment:
Can you bump to 1.0.0? Stable charts should be >1.0 and we stayed too long in 0.x

Reply:
Sure thing.